### ISSUE TITLE
Should `.description` be shown in “Description” column

### ISSUE THREAD

**Issue by wooorm** [steelbrain/linter-ui-default#205]
Hi! 👋

 [`upgrading-to-standard-linter-v2.md`](https://github.com/steelbrain/linter/blob/master/docs/guides/upgrading-to-standard-linter-v2.md#indexjs) states the following:

> `html` is no longer allowed, similar functionality is available as `description` which is either a string or a function (Promise) returning a string which is then rendered as Markdown.

Maybe I’ve missed something, but when upgrading I thought I could use `.description` for rich (non plain-text) message reasons. Especially as there’s now a column, “Description”, which apparently [does not use `.description` on messages](https://github.com/steelbrain/linter-ui-default/blob/9f2680f50c40fc634789dc741b9026a35e2b9609/lib/panel/component.js#L105-L112), to my surprise :)

#### TLDR

Possible bug: should `description` be used in `linter-ui-default`
Question: if not, where does it end up?

**Comment by steelbrain** [comment]
Question is actually used in linter-ui-default, you have to expand it on the tooltips, we don't show it on the linter panel because description is by nature YUGE and panel doesn't have as much space.

You can click the carot on the left most of the tooltip or you can bind `linter-ui-default:expand-tooltip` command to a key of your preference

**Comment by wooorm** [comment]
Oh okay, sorry, I missed that. I thought that, due to the suggestion of using `description` instead of `html` they’d show up in the same places.

Okay, one final question! Is it somehow possible to use markup/markdown in `excerpt`?

**Comment by steelbrain** [comment]
There's currently no way to add markdown/markup to excerpt, the reason is that we want it to be text and consistent text, for example `linter-eslint` will change from the ribbon style stuff into simple `rule: message`. It previously used to add a link into the message html, but now we've added a separate `url` property for that! :)

**Comment by wooorm** [comment]
Alright, thanks a lot for your help!
